utils/auth/function.ts

```
@@ -1,9 +1,147 @@
+});
+
+export const signInWithGoogle = async (showToast: (type: TOAST, message: string) => void) => {
+  try {
+    await GoogleSignin.hasPlayServices();
+    const response = await GoogleSignin.signIn();
+    if (isSuccessResponse(response)) {
+      const { idToken, user } = response.data;
+
+      const { data, error } = await supabase.auth.signInWithIdToken({
+        provider: "google",
+        token: idToken!!,
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)